### PR TITLE
feat(docs): add ```Alert``` theming docs

### LINKED ISSUE #1036: Theming: Alert
### Subject

Alert

### Description

Create a basic theming documentation for the `Alert` component that showcases how you can style it.

Theming file: [Alert theming file](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/alert/theming.mdx)

### Template Reference

For single part components, feel free to use the `Button` component theming doc as a template reference: [Button theming](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/button/theming.mdx).

For multipart components, feel free to use the `Input` component theming doc as a template reference: [Input theming](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/input/theming.mdx).

### PR DESCRIPTION
**Pull request by HariVP03**
Closes #1036 

## 📝 Description

> Adds the theming docs for the `Alert` component

## ⛳️ Current behavior (updates)

> There are no theming focs for the `Alert` component currently

## 🚀 New behavior

> Added the theming docs for the `Alert` component

## 💣 Is this a breaking change (Yes/No):

> No

### CODE REVIEW

File: configs/sandpack-contents/component-theming/alert.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+const baseStyle = definePartsStyle({
+  // define the part you're going to style
+  title: {
+    color: "teal.500", // change the color of the title text of the alert
+  },
+});
+
+// Defining a custom variant called mono
+const mono = definePartsStyle((props) => {
+  const { colorScheme: c } = props;
```
Comment (nikolovlazar) [inline configs/sandpack-contents/component-theming/alert.js:79]: You can obtain the `status` from the props here, and color the alert based on it. Try doing `console.log(props)` to see it. I wouldn't use a `colorScheme` in this case. The `status` should control the color of the alert.

File: configs/sandpack-contents/component-theming/alert.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  <ChakraProvider theme={theme}>
+    <App />
+  </ChakraProvider>
+);`,
+  AlertTheme: `import { alertAnatomy as parts } from "@chakra-ui/anatomy";
+import {
+  createMultiStyleConfigHelpers,
+  defineStyle,
+} from "@chakra-ui/styled-system";
+import { extendTheme } from "@chakra-ui/react";
```
Comment (nikolovlazar) [inline configs/sandpack-contents/component-theming/alert.js:65]: You're not using this import.

File: content/docs/components/alert/theming.mdx [hunk 3]
```diff
@@ -2,3 +2,186 @@
...
 scope: theming
 ---
+
+The `Alert` component is a multipart component. The styling needs to be applied
+to each part specifically.
+
+> To learn more about styling multipart components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-multipart-components)
+> page.
+
```
Comment (nikolovlazar) [inline content/docs/components/alert/theming.mdx:12]: Can you put the `## Anatomy` section here with its parts? I'll prepare a custom illustration for you afterwards.

**Review by nikolovlazar — CHANGES_REQUESTED** [review]

File: content/docs/components/alert/theming.mdx [hunk 4]
```diff
@@ -2,3 +2,206 @@
...
+<AspectRatio ratio={16 / 9}>
+  <svg
+    width='100%'
+    height='100%'
+    viewBox='0 0 480 270'
+    fill='none'
+    xmlns='http://www.w3.org/2000/svg'
+  >
+    {/* SVG here */}
```
Comment (nikolovlazar) [inline content/docs/components/alert/theming.mdx:29]: Illustration source: https://gist.github.com/nikolovlazar/c2fedc96b3d3aa19a1a7a8814c7437ae

File: configs/sandpack-contents/component-theming/alert.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+      border: "1px solid",
+      borderColor: "gray.100",
+      bg: status === "error" ? "red.800" : "gray.100",
+    },
+    title: { fontFamily: "mono" },
+    description: {
+      fontFamily: "mono",
+    },
+  };
+});
```
Comment (nikolovlazar) [inline configs/sandpack-contents/component-theming/alert.js:95]: A couple of changes:
- remove the `console.log` statement
- can you redesign this? the `red.800` and `gray.800` don't play well together 😅

**Review by nikolovlazar — CHANGES_REQUESTED** [review]

File: configs/sandpack-contents/component-theming/alert.js (hunk 5 above)
Comment (HariVP03) [reply configs/sandpack-contents/component-theming/alert.js:95]: Sure sure

Comment (HariVP03) [reply configs/sandpack-contents/component-theming/alert.js]: @nikolovlazar  I've removed the ```console.log```. How is this design? I tried to do something different by having the status indicated by the alert's ```borderColor```

**Review by nikolovlazar — APPROVED** [review]

**Pull request merged**